rewrite_with_pygame_solo_version.py

```diff
@@ -0,0 +1,231 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+Created on Thu May 28 05:15:43 2020
+
+@author: carolinstreitberger
+"""
+######## Imports
+import numpy as np
+import pygame
+import sys
+import random
+# from pygame.locals import * # not sure if I need this
+
+######## The first step is to create the grid of the computer
+def generate_random_position(size_ship):
+    """
+    This function will generate the position of a ship of a given size.
+
+    Parameters
+    ----------
+    size_ship : TYPE #?
+        Size of the ship whose position will be determined.
+
+    Returns
+    -------
+    orientation : TYPE
+        Vertical or horizontal.
+    y_start_position : TYPE
+        Starting point of the ship on the y-axis.
+    x_start_position : TYPE
+        Starting point of the ship on the x-axis.
+    """
+    orientation = random.randint(0, 1) # 0 is vertical and 1 is horizontal orientation
+    if (orientation == 0):
+        y_start_position = random.randint(1, CELLS - size_ship)
+        # 10 - size_ship so that the whole ship is never bigger than 10
+        x_start_position = random.randint(1, CELLS)
+    elif (orientation == 1):
```

Comment:
Here you can just say `else` as your only options are 0 (vertical) and 1 (horizontal). Your game should not allow any other orientation :) You can aply the same logic throughout your code. Generally, `elif` is used wwhen there are more than 2 alternatives.